<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 255, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="NoButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 92, 92); 
font: 75 13pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Против</string>
    </property>
   </widget>
   <widget class="QListWidget" name="NoList">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ZaList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZaButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(131, 255, 124); 
font: 75 13pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>За</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 129, 90); 
font: 75 13pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LabelLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">backgroud-color: rgb(255, 255, 255); font: 75 14pt &quot;MS Serif&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="VoiceLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">backgroud-color: rgb(255, 255, 255); 
font: 75 13pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
